build.gradle

```diff
@@ -416,6 +418,7 @@ subprojects.each { subproject ->
 apply plugin: 'org.jetbrains.intellij'
 
 intellij {
+  instrumentCode false
```

Comment:
We don't have any instrumentable (i.e. Java) code in top level module so far... but that's gonna change with e.g. PR #273